### PR TITLE
Add MSI version definition.

### PR DESCRIPTION
**Pull request by JessieF**
PTF

### CODE REVIEW

**Comment by JessieF** [comment]
PTF

File: src/build.cmd [hunk 1]
```diff
@@ -1,7 +1,13 @@
 :: Copyright (c) Microsoft. All rights reserved.
 :: Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+:: build PTF -- call build.cmd
```
Comment (yazeng) [inline src/build.cmd]: we cannot add build version here, right?

File: src/htmltestlogger/HtmlTestLogger.csproj [hunk 2]
```diff
@@ -71,7 +71,7 @@
   </ItemGroup>
   <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.targets" />
   <Target Name="AfterBuild">
-    <Copy SourceFiles="$(TargetDir)$(TargetFileName)" DestinationFolder="..\Bin\ptf\HtmlTestLogger\" />
-    <Copy SourceFiles="$(TargetDir)$(TargetName).pdb" DestinationFolder="..\Bin\ptf\HtmlTestLogger\" />
+    <Copy SourceFiles="$(TargetDir)$(TargetFileName)" DestinationFolder="..\..\drop\ProtocolTestFramework\Bin\HtmlTestLogger\" />
```
Comment (yazeng) [inline src/htmltestlogger/HtmlTestLogger.csproj]: use PTF_Root for the destinationfolder

File: src/build.cmd (hunk 1 above)
Comment (JessieF) [inline src/build.cmd]: Yes

**Pull request merged**